frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/top-consumers-card.tsx

```diff
@@ -0,0 +1,202 @@
+          <Chart
+            domain={{ y: [0, 1.5 * maxCapacityConverted.value] }}
+            height={175}
+            padding={{ top: 20, bottom: 20, left: 20, right: 20 }}
+            containerComponent={
+              <ChartVoronoiContainer
+                labels={(datum) => `${datum.y} ${maxCapacityConverted.unit}`}
+                labelComponent={<ChartTooltip style={{ fontSize: 8, padding: 5 }} />}
+              />
+            }
+            themeColor={ChartThemeColor.multi}
+            scale={{ x: 'time' }}
+          >
+            <ChartAxis
+              tickFormat={(x) => formatToShortTime(x)}
+              style={{ tickLabels: { fontSize: 8, padding: 5 } }}
+            />
+            <ChartAxis
+              dependentAxis
+              tickValues={getYTickValues(maxCapacityConverted.value)}
+              style={{ tickLabels: { fontSize: 8, padding: 5 }, grid: { stroke: '#4d525840' } }}
+            />
+            <ChartGroup>{chartLineList}</ChartGroup>
+          </Chart>
+          <ChartLegend
+            x={10}
+            y={5}
+            data={legends}
+            themeColor={ChartThemeColor.multi}
+            orientation="horizontal"
+            symbolSpacer={7}
+            height={30}
+            gutter={10}
+            style={{
+              labels: { fontSize: 8 },
+            }}
+          />
+        </>
+      );
+    }
+    return (
+      <EmptyState className="graph-empty-state" variant={EmptyStateVariant.full}>
+        <EmptyStateIcon size="sm" icon={ChartLineIcon} />
+        <Title size="sm">No Prometheus datapoints found.</Title>
+      </EmptyState>
+    );
+  },
+);
+
+const metricTypes = _.keys(TopConsumerResourceValue);
+const sortByTypes = _.keys(TopConsumerSortByValue);
+
+const metricTypesOptions = _.zipObject(metricTypes, metricTypes);
+const sortByOptions = _.zipObject(sortByTypes, sortByTypes);
+
+const TopConsumerCard: React.FC<DashboardItemProps> = ({
+  watchPrometheus,
+  stopWatchPrometheusQuery,
+  prometheusResults,
+}) => {
+  const [metricType, setMetricType] = React.useState(metricTypes[0]);
+  const [sortBy, setSortBy] = React.useState(sortByTypes[0]);
+  React.useEffect(() => {
+    const query =
+      TOP_CONSUMER_QUERIES[
+      StorageDashboardQuery[TopConsumerResourceValue[metricType] + TopConsumerSortByValue[sortBy]]
+      ];
+    watchPrometheus(query);
+    return () => stopWatchPrometheusQuery(query);
+  }, [watchPrometheus, stopWatchPrometheusQuery, metricType, sortBy]);
+
+  const topConsumerstats = prometheusResults.getIn([
+    TOP_CONSUMER_QUERIES[
+    StorageDashboardQuery[TopConsumerResourceValue[metricType] + TopConsumerSortByValue[sortBy]]
+    ],
+    'result',
+  ]);
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Top Consumers</DashboardCardTitle>
+        <div className="ceph-top-consumer-card__dropdown">
```

Comment:
height of the card header is inconsistent with others. Take a look at these styles https://github.com/openshift/console/pull/1722/files#diff-163991d4c119db922a729100ce2bea7fR91

Reply:
sorry, those dropdowns are actually not in header as yours are.
@andybraren @matthewcarleton any help on how to include these dropdowns into PF4 Card Header and keep header height consistent ? 

![dropdowns-header](https://user-images.githubusercontent.com/2078045/61385674-1ff0d500-a8b3-11e9-853a-4b00b69d59a3.png)

See the difference between Capacity and Top Consumers header

Reply:
Hey @rawagner I believe you can use the actions area for the card header provided. That should fix the spacing isssues. If not we may need to investigate further if there is unnecessary spacing added to other elements in there.